types/multiselect/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for multiselect 0.9
+// Project: https://github.com/lou/multi-select
+// Definitions by: nagamejun <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/multiselect/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me